src/ms.tex

```diff
@@ -1379,24 +1388,19 @@ \subsection{A roadmap for future priorities}
-be on a paid plan; additionally NumFOCUS has negotiated a special rate for
-open-source projects. While the Slack and Astropy-dev mailing lists are primarily used to discuss the project direction and updates, it was noted that the use of a Facebook community could present a barrier to open source. The Astropy Project identified a need for a public, archived, searchable, and easy-to-navigate interface for users to ask for help. Accordingly, we have commissioned
-a Discourse site which is more open than the Facebook group and more user-oriented
-than Astropy Slack. A benefit of the Moore Foundation grant is that Astropy
-developers are able to invoice as independent contractors the time they
-spend helping users on these forums.
-\end{itemize}
-
-\begin{itemize}
-\item Attracting new contributors when the code has become quite complex,
-\item Contributor to maintainer mentoring,
-\item Long-term / sustained funding for maintaining infrastructure,
-\item ...
-\end{itemize}
+Over the last decade, the needs of (and challenges faced by) the \astropy
+Project have evolved, but many common themes persist.
+For example, ...list...
+In many cases, these themes are not unique to \astropy: Many Open-Source
+Software communities are facing and addressing similar issues.
+Below we discuss a few of these challenges and ...
+
+\begin{description}
+  \item[Supporting an active group of maintainers] With funding, can pay maintainers. But: time commitment, some positions don't allow. Need for new career paths on the spectrum from software engineer to astronomy researcher. Also need to attract new non-hired contributors to sustain the project, but challenging when the code has become quite complex (astropy as an "institution", don't want to be a walled-off group). (Need for fast feedback and code reviews to empower contributors, but puts a burden on maintainers)
+  \item[Diversity of contributor, maintainer, coordinator communities] Practical issues: translation of content and documentation, github is not accessible in some countries.
+  \item[Long-term / sustained funding for maintaining infrastructure] Further from "science," still critical to astropy, but who will provide updates and maintenance? More than just updating versions and releasing. (grants last 3 years; infrastructure evolves, nature of open source ecosystem)
+  \item[Understanding, communicating with, and engaging our user communities] Different overlapping and intersecting communities of users: telescopes, archives, researchers, students, educators, general public. Communities have different needs from Astropy. Also, as ecosystem grows, making sure users know about all of the functionality (workshops, tutorials, educational materials, discoverability, accessibility) (Continued engagement with user community).
```

Comment:
@nmearl would you be willing to write a few sentences to address these issues?

Reply:
I may need a day or two, but I will get to this!

Reply:
I have some time today, so I'll work on this